DDDEastAnglia/Controllers/HomeController.cs

```diff
@@ -56,6 +56,11 @@ public ActionResult CodeOfConduct()
         {
             return View();
         }
+        
+//        public ActionResult Environmental()
```

Comment:
This correct?

Reply:
yeah - still working on it and stupidly started the images on the same fork.
Images are higher priority for now. So keeping the Env stuff dark for now.

Little bit lazy - should have split it all up in to sep change sets!

Reply:
understood.. just checking it wasn't a mistake